man/fi_usnic.7.md

```diff
@@ -98,17 +98,30 @@ request a version rather than using the header provided
 `FI_EXT_USNIC_INFO_VERSION`. Although there is a version 1 of the extension,
```

Comment:
On line 91, do we need to say "FI_USNIC_FABRIC_OPS_1 or FI_USNIC_FABRIC_OPS_2 ..."?